_news-items/2025-04-16-mep.html

```diff
@@ -0,0 +1,14 @@
+---
+# Copyright 2025 seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: CC-BY-SA-4.0
+title: MEP joins the seL4 Foundation
+anchor: member-mep
+---
+
+<p>
+    <a href="https://www.mep-info.com/">
+    <img src="../Foundation/Membership/LOGOS/mep.svg"
+       style="width: 20%; margin-right:20px; float: right;" alt="MEP logo">
+    </a>
+    The seL4 Foundation is pleased to welcome <a href="https://www.mep-info.com/">MEP</a> as a <a href="../Foundation/Membership">Member</a>. MEP is a voice communications supplier for Air Traffic and Maritime critical communications worldwide. We are using seL4 in the heart of our Voice Communication System, the SureVoice Solid. MEP has chosen for process separation within the seL4 to achieve 24/7 availability.
+</p>
```

Comment:
This last sentence is broken ("has chosen" what?). I also think we should avoid "we" to avoid confusion between Foundation and MEP. I know the text comes from MEP, but I suggest the following tweaks to the last 2 sentences (you can check with them if you want):

Its Voice Communication System, the SureVoice Solid, runs on seL4 at its heart. MEP has chosen seL4 for process separation to achieve 24/7 availability.

I would also add a link to https://www.mep-info.com/products/surevoice-solid

Reply:
Thanks! Nice catch - I completely missed the missing word.